app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -68,7 +68,7 @@ def total_count
       end
 
       def hosts
-        job_invocation.targeting.hosts.order(:name, :id)
+        job_invocation.targeting.hosts
```

Comment:
Will this work with the batches? With randomized order, I guess every batch could include some hosts that were already planned for. One option to fix this could be to exclude the hosts that were already planned from this search, instead of using paging for moving though the hosts list

Reply:
I run a test job with randomized order on 500 hosts where each host would write its name into a shared file, it seems to work just fine.

```shell
curl https://paste.pound-python.org/raw/6NcIyuj45nDXLhuXTvjK/ | sort | uniq -d
```

Reply:
Guess the question is if we can rely on a db always reading the join table records in the order in which they were saved

Reply:
Yes, I would be more confident if param we're basing the ordering on (even if it was the id of the `id` of the `targeting_hosts` table.